textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/WordBase.java

```diff
@@ -0,0 +1,35 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashSet;
+
+/**
+ * Created by shiladityasen on 4/30/16.
+ * Class to provide simple abstraction of a dictionary of words with constant retrieval time
+ * Used by FuzzyTokenizer to process query
+ */
+public class WordBase
```

Comment:
Just want to add an additional note: in general, a dictionary used to tokenize a string may also include the frequency of each word so that a dynamic programming algorithm can find the best tokenization to maximize the likelihood.  In that sense, the dictionary used here is slightly different from the dictionary used in the "DictionaryMatcher".

For now it's OK to use Team1's dictionary for this QueryRewriter operator.